template/src/lib/sdk/widgets/IsochroneWidget.js

```diff
@@ -0,0 +1,174 @@
+import {
```

Comment:
Move this as a view inside of Stores

Reply:
As a view? are you sure? I think that we can re-use it like geocodingWidget.
If you want to move it outside of lib I think that should be to common components.